site/content/docs/contributing/getting-started.md

```diff
@@ -73,10 +67,22 @@ docker --version
 ```
 This documentation is written using Docker version 18.09.2.
 
+### Install Go (optional)
+
+KIND is written in [Go][golang], however our makefiles automatically ensure the
+correct version of go when building or testing.
+
+You may still wish to install go on your machine to make it easier to integrate
```

Comment:
go here is inconsisten, you use Go in other places

Reply:
I think we should use it as `Go` when first talking about it so it's clear that it's a proper noun, but it's usually just written `go`.